src/anm/analytics.js

```diff
@@ -33,40 +43,41 @@ var Analytics = function () {
     this.trackPlayingComplete = this.trackPlayer('playing_complete');
 };
 
-Analytics.prototype.track = function track(name, opts) {
-    opts = opts || {};
-    opts.name = name;
-    opts.referer = document.referrer;
-    opts.lang = navigator.language || navigator.userLanguage;
-    opts.url = location.href;
-    opts.screenHeight = screen.height;
-    opts.screenWidth = screen.width;
-    opts.windowHeight = window.innerHeight;
-    opts.windowWidth = window.innerWidth;
-    opts.timestamp = new Date().getTime();
+Analytics.prototype.track = function track(name, player, action) {
     if (this.enabled) {
-        this.queue.push(opts);
+        this.queue[player.viewId] = this.queue[player.viewId] || {
+                viewId: player.viewId,
+                projectId: player.anim.meta._anm_id,
+                referer: document.referrer,
+                lang: navigator.language || navigator.userLanguage,
+                url: location.href,
+                screenHeight: screen.height,
+                screenWidth: screen.width,
+                windowHeight: window.innerHeight,
+                windowWidth: window.innerWidth,
```

Comment:
>  On a desktop browser, often the performance impact seems to be relatively unnoticeable. The user impact at a 10ms reflow time is rather negligible. However, herein seems to be much of the performance loss on mobile devices. The same pages that take only 10ms to reflow on a desktop can take over 1000ms on a mobile device. Every single change to the DOM further impacts that time (this is part of the reason why JavaScript-based animation looks so terrible on mobile devices).

I can't say player does no reflows at all, on the other hand, but if new ones can be avoided, why not do it one time for a page, not for a player?